surveys/sphinx-survey-2025.yml

```diff
@@ -14,81 +14,70 @@
+    Open text field questions may be quoted, but will not be shown in full.
 
-    Results will be public with identifying information removed. Open text
-    field questions may be quoted, but will not be shown in full.
+    This survey is run by the Sphinx community and [Quansight Labs]; survey
+    data will be deleted from any private accounts once made public. If you
+    have any questions, contact Tania Allard (Director of Quansight Labs)
+    at tallard@quansight.com and mention Sphinx Survey 2025 in the subject line.
 
-    This survey is run by the Sphinx community and Quansight Labs; survey 
-    data will be deleted from any private accounts once made public. If you 
-    have any questions, reach out to tallard@quansight.com and mention 
-    Sphinx Survey 2025 in the subject line.
+    [Sphinx]: https://www.sphinx-doc.org/
+    [Quansight Labs]: https://quansight.com/labs/
 
-- title: "Tell us about you"
-  description: >-
-    First, we'd like to know about how you work and what tools you use.
-  type: statement
-
-# Motivation: Understand how the responder views themself in relation to Sphinx.
+# Motivation: Understand how the responder views themselves in relation to Sphinx.
 - title: "What is your relation to the Sphinx project?"
   multiple: true
   choices:
   - "I read documentation that is built with Sphinx"
   - "I use Sphinx to build documentation"
-  - "I build extensions for Sphinx"
-  - "I maintain a Sphinx theme"
-  - "I contribute to Sphinx"
+  - "I maintain or have contributed to a Sphinx extension"
+  - "I maintain or have contributed to a Sphinx theme"
+  - "I have contributed to the Sphinx project"
   - "None of the above"
 
 # Motivation: Understand the responder's self-reported background and skills.
-- title: "Which of the following roles is closest to your job?"
+- title: "Which of the following best describes your occupation?"
   other: true
   choices:
   - "Academic researcher"
   - "Data scientist"
   - "Developer / Programmer"
+  - "Executive / Manager"
   - "ML engineer / MLOps"
   - "Product manager"
   - "QA engineer"
+  - "Student (in full-time education)"
   - "Technical support"
   - "Technical writer"
   - "Prefer not to answer"
 
 # Motivation: Understand the responder's self-reported background and skills.
-- title: "Do you most often work on a team or independently?"
+- title: "Do you most often work in a team or independently?"
   other: true
   choices:
   - "I most often work on projects independently"
   - "I most often work on projects as part of a team"
-  - "I spend about equal time working on projects independently as I do on
-    a team"
+  - "I spend about equal time working on projects independently as I do on a
+    team"
   - "Prefer not to answer"
 
 # Motivation: Understand the responder's self-reported background and skills.
-- title: "How many years have you been using Python for?"
+- title: "How many years of professional experience do you have?"
```

Comment:
Cf discussion in https://github.com/orgs/sphinx-doc/discussions/13331#discussioncomment-12794511 on removing Python-specific-ness.

Reply:
Do we care about the kind of professional experience? If so, how can we qualify this? "Python" was specific "programming" would be more general, but not everybody programs. "documentation-related" would be an option but that may be too strong. 🤷

Reply:
I do not believe we care much about the "type"